### PR TITLE
Adds !mentorwho as a discord command

### PR DESCRIPTION
**Pull request by swissloaf**
Adds mentorwho

🆑
add: People can do !mentorwho and get online mentors
/🆑

### CODE REVIEW

**Comment by swissloaf** [comment]
> you need to make a world topic handler on the server-side for mentorwho

i literally dmed u and you never responded

**Comment by swissloaf** [comment]
adding world topic hold on do not merge

File: models/Discord/Commands/DiscordCommandMentorWho.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const StringUtils = require('../../Utils/String.js');
+
+class DiscordCommandMentorWho extends DiscordCommand {
+
+  constructor(subsystem) {
+    super("mentorwho", "Get current mentors online.", 'mentorwho', subsystem);
+  }
+
+  onRun(message, permissions, args) {
+    var config = this.subsystem.manager.getSubsystem("Config").config;
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandMentorWho.js:11]: 🤔

Comment (swissloaf) [reply models/Discord/Commands/DiscordCommandMentorWho.js:11]: do tell please

**Comment by Codeatmos** [comment]
take everything back,mins good.

**Comment by swissloaf** [comment]
> you need to make a world topic handler on the server-side for mentorwho

done, needs merging

**Comment by Hopekz** [comment]
Not sure if outside the scope of this PR but you should add this information to !info as well in my opinion.

**Comment by swissloaf** [comment]
> Not sure if outside the scope of this PR but you should add this information to !info as well in my opinion.

i was thinking about that, lemme see what i can do

**Comment by swissloaf** [comment]
> Not sure if outside the scope of this PR but you should add this information to !info as well in my opinion.

dont know if i can, nich specifically said in the Info command that admins should be the last line due to its dynamic size. not sure how that would mix with mentors who would also be dynamic size

**Comment by Hopekz** [comment]
> > Not sure if outside the scope of this PR but you should add this information to !info as well in my opinion.
> 
> dont know if i can, nich specifically said in the Info command that admins should be the last line due to its dynamic size. not sure how that would mix with mentors who would also be dynamic size

Nich knows more about that then me.
If he says something is preventing it I trust his judgement.
Would be nice if it was possible though.

File: models/Discord/Commands/DiscordCommandMentorWho.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    var byondConnector = this.subsystem.manager.getSubsystem("Byond Connector").byondConnector;
+    var byondmessage = "?mentorwho";
+
+    }
```
Comment (monster860) [inline models/Discord/Commands/DiscordCommandMentorWho.js:15]: you have a stray closing curly brace here

File: models/Discord/Commands/DiscordCommandMentorWho.js (hunk 1 above)
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandMentorWho.js:11]: 🤔

Comment (swissloaf) [reply models/Discord/Commands/DiscordCommandMentorWho.js:11]: you gotta say something more than the thinking emoji

**Pull request merged**